samples/jenkins/README.md

```diff
@@ -1,5 +1,373 @@
+
+### Development pipeline
+The pipeline, that trains the model, promotes it and export assets as a Jenkins artifact consists of stages:
+1. Install cdpctl - downloads and installs the latest `cdpctl` version using [install_cpdctl.sh](scripts/helpers/install_cpdctl.sh) script
+2. Train model - trains the model using notebook and datasets located in Watson Studio with the [train_model.sh](scripts/dev/train_model.sh) script 
+3. Promote to DEV - executes [promote_to_dev_space.sh](scripts/dev/promote_to_dev_space.sh) to promote trained model from the 
+project to the development deployment space
+4. Export assets - executes [export_dev_space.sh](scripts/dev/export_dev_space.sh) to export assets from the development deployment space to an archive, 
+then, using `archiveArtifacts` step, attach the archive to the Jenkins build as a build artifacts.
+5. Cleanup - executes [cleanup.sh](scripts/dev/cleanup.sh) to delete all models, assets and jobs from the project.
+
+### Test pipeline
+Import of the artifacts from the previous job, model deployment and evaluation is defined in the test pipeline with the following stages:
+1. Install cdpctl - same as in the development pipeline 
+2. Import assets - using `copyArtifacts` step downloads artifacts archived in development pipeline, then imports the archive to 
+the newly created deployment space with [create_test_space.sh](scripts/test/create_test_space.sh) script.
+3. Deploy model - deploys model with [create_model_batch_deployment.sh](scripts/test/create_model_batch_deployment.sh) script.
+4. Evaluate model - evaluates model deployment using [create_script_batch_deployment.sh](scripts/test/create_script_batch_deployment.sh) script.
+5. Cleanup - executes [cleanup.sh](scripts/test/cleanup.sh) to delete delete whole space with all assets.
+
+### Production pipeline
+Same as previously, the preparation part defines the default execution node and loads the secret file. Rest 
+1. Install cpdctl - same as in the development pipeline
+2. Import assets - update existing AI assets by importing them from the archive stored in the artifacts repository using 
+[import_to_prod_space.sh](scripts/prod/import_to_prod_space.sh) script.
+3. Update model deployment - executes [update_model_deployment.sh](scripts/prod/update_model_deployment.sh) script to redeploy the new improved model.
+4. Evaluate the model - run regression tests to ensure the quality of the new model version using [run_model_deployment_validation.sh](scripts/prod/run_model_deployment_validation.sh) script, then attach the evaluation
+results to the Jenkins job with `archiveArtifacts` step.
+5. Update evaluation script deployment - redeploy the evaluation script using [update_script_deployment.sh](scripts/prod/update_script_deployment.sh) script.
+
+## Run the scenario
+When all configuration steps are done, there is a time to execute the scenario and look at the result.
```

Comment:
~~there is a time~~ **it's time** to execute the scenario and look at the **results**.

Reply:
Fixed.